src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/GetAssignmentController.java

```diff
@@ -58,7 +58,8 @@ public GetAssignmentController(@Autowired RetrieveRoleAssignmentOrchestrator ret
     @LogAudit(operationType = GET_ASSIGNMENTS_BY_ACTOR,
         size = "T(uk.gov.hmcts.reform.roleassignment.util.AuditLoggerUtil).sizeOfAssignments(#result)",
         actorId = "T(uk.gov.hmcts.reform.roleassignment.util.AuditLoggerUtil).getActorIds(#result)",
-        correlationId = "#correlationId")
+        correlationId = "#correlationId",
+        responseTime = "T(uk.gov.hmcts.reform.roleassignment.util.AuditLoggerUtil).responseTime(#result)")
     public ResponseEntity<RoleAssignmentResource> retrieveRoleAssignmentsByActorId(
```

Comment:
this is not require. irrespective of this tag you have to add responseTime in audit log